tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_sso_security_manager.py

```diff
@@ -59,7 +61,9 @@ def oauth_user_info(self, provider, response=None):
         if provider == "openedxsso":
             user_profile = self.decoded_user_info()
 
-            user_roles = self._get_user_roles(user_profile.get("preferred_username"))
+            language_preference = request.cookies.get('openedx-language-preference', 'en')
```

Comment:
I think this forces a requirement for the superset server to be on the same domain as the LMS, which would probably break things like preset.io Superset (not that it works now, but it would create another barrier). Are the language prefs not available in any of the LMS calls we're currently making? I think it might be preferable to add one more API call to get them if necessary.

Reply:
Yeah, I was thinking in calling the account API instead